products/jbrowse-desktop/package.json

```diff
@@ -102,7 +102,7 @@
     "webpack": "^5.72.0"
   },
   "devDependencies": {
-    "electron": "23.1.1"
+    "electron": "^23.1.1"
```

Comment:
we actually pin this without semver, because elsewhere in the package.json there is an 'electronVersion' field that should sync up with this

Reply:
Will remove this

Reply:
@cmdcolin I have resolved this